features/provider/dashboard/tabs.feature

```diff
@@ -0,0 +1,36 @@
+Feature: Dashboard search bar
+  In order to navigate easily to products and backends
+  As a provider
+  I want to be able to filter them by name
+
+  Background:
+    Given a provider is logged in
+    And a service "My Fancy Product"
+    And a service "My Regular Product"
+    And a backend api "My Fancy Backend API"
+    And a backend api "My Regular Backend API"
+    And I go to the provider dashboard
+
+  @javascript
+  Scenario: Filtering products
+    When I select the products tab
+    And I should see "My Fancy Product" in the apis dashboard products tabs section
```

Comment:
Perhaps we can improve this later with a more concise table of what should be seen, but good for now.